src/components/TrackTile/TrackerDetails/coerce-to-nonnegative-value.ts

```diff
@@ -8,5 +8,5 @@ export const coerceToNonnegativeValue = (num: string, defaultNum: number) => {
     return defaultNum;
   }
 
-  return Math.round(targetNum);
```

Comment:
I don't see any reason why we would need to be rounding this. Perhaps we did not allow fractional targets in the past.

Reply:
@sternetj do you have any recollection?  As long as we have display logic to ensure we never show something like 1.3333333, I agree this should be fine.

Reply:
I can't think of anywhere which we compute these values in a way that could result in that many digits.

Reply:
I bet this was to handle when we had a number input for selecting your target. A user could enter something like `13e-1` which is a valid JS number but not a whole number.

It seems like it is still used on the tracker value. Was this causing you problems @bruce-glazier

Reply:
We allow for selecting values such as 0.25, 0.5, 0.75 for units like hours and we use this function to convert from string to number for storing the value. It was rounding those to the nearest whole number.

Reply:
Gotcha, that makes sense. Sounds like its safe to remove